build_aarch64_wheel.py

```diff
@@ -82,121 +145,151 @@ def wait_for_connection(addr, port, timeout=5, attempt_cnt=5):
+
     git_clone_flags = " --depth 1 --shallow-submodules" if shallow_clone else ""
     print('Configuring the system')
-    update_apt_repo(addr)
+    if not host.using_docker():
+        update_apt_repo(host)
+        host.run_cmd("sudo apt-get install -y ninja-build g++ git cmake gfortran unzip")
+    else:
+        host.run_cmd("yum install -y sudo")
+        host.run_cmd("conda install -y ninja")
 
-    run_ssh(addr, "sudo apt-get install -y ninja-build g++ git cmake gfortran unzip")
     if not use_conda:
-        run_ssh(addr, "sudo apt-get install -y python3-dev python3-yaml python3-setuptools python3-wheel python3-pip")
-    run_ssh(addr, "pip3 install dataclasses typing-extensions")
+        host.run_cmd("sudo apt-get install -y python3-dev python3-yaml python3-setuptools python3-wheel python3-pip")
+    host.run_cmd("pip3 install dataclasses typing-extensions")
     # Install and switch to gcc-8 on Ubuntu-18.04
-    if ami == ubuntu18_04_ami and compiler == 'gcc-8':
-        run_ssh(addr, "sudo apt-get install -y g++-8 gfortran-8")
-        run_ssh(addr, "sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 100")
-        run_ssh(addr, "sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-8 100")
-        run_ssh(addr, "sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-8 100")
+    if not host.using_docker() and host.ami == ubuntu18_04_ami and compiler == 'gcc-8':
+        host.run_cmd("sudo apt-get install -y g++-8 gfortran-8")
+        host.run_cmd("sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 100")
+        host.run_cmd("sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-8 100")
+        host.run_cmd("sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-8 100")
     if not use_conda:
         print("Installing Cython + numpy from PyPy")
-        run_ssh(addr, "sudo pip3 install Cython")
-        run_ssh(addr, "sudo pip3 install numpy")
-    # Build OpenBLAS
-    print('Building OpenBLAS')
-    run_ssh(addr, f"git clone https://github.com/xianyi/OpenBLAS -b v0.3.10 {git_clone_flags}")
-    # TODO: Build with USE_OPENMP=1 support
-    run_ssh(addr, "pushd OpenBLAS; make NO_SHARED=1 -j8; sudo make NO_SHARED=1 install; popd")
-
-    # Build FFTW
-    print("Building FFTW3")
-    run_ssh(addr, "sudo apt-get install -y ocaml ocamlbuild autoconf automake indent libtool fig2dev texinfo")
-    # TODO: fix a version to build
-    # TODO: consider adding flags --host=arm-linux-gnueabi --enable-single --enable-neon CC=arm-linux-gnueabi-gcc -march=armv7-a -mfloat-abi=softfp
-    run_ssh(addr, f"git clone https://github.com/FFTW/fftw3 {git_clone_flags}")
-    run_ssh(addr, "pushd fftw3; sh bootstrap.sh; make -j8; sudo make install; popd")
+        host.run_cmd("sudo pip3 install Cython")
+        host.run_cmd("sudo pip3 install numpy")
+
+    build_OpenBLAS(host, git_clone_flags)
+    # build_FFTW(host, git_clone_flags)
```

Comment:
are we disabling fftw?

Reply:
Because it is not used from the codebase at the moment, is it?